lib/state/appState.js

```diff
@@ -89,6 +89,17 @@ appState.getInitial = function() {
   };
 };
 
+appState.hideUnavailableDevices = function() {
+  var currentOS = this.app.device._os; // "mac", "win", "android", "cros", "linux", or "openbsd"
+
+  if(currentOS === 'mac') {
+    this.app.state.uploads = _.reject(this.app.state.uploads, function(d){
+      return d.key === 'precisionxtra'; // TODO: || d.key === 'abbottfreestylelite' || d.key === 'abbottfreestylefreedomlite'
```

Comment:
I think the list of unsupported devices per OS will be quite short. So far it's just the Abbott devices that turned out to have a very different Mac implementation, so I'm hoping that we will be able to supported both Mac and Windows for each new device that we roll out.

If this turns out not to be the case, I agree that it's better to store it as an attribute in `uploads`.